data/disney

```diff
@@ -47,10 +47,18 @@ disney.sg
 20thcenturystudios.com.au
 20thcenturystudios.com.br
 20thcenturystudios.jp
+adobedtm.com
 adventuresbydisney.com
 babble.com
 babyzone.com
+bamgrid.com
+bam.nr-data.net
 beautyandthebeastmusical.co.uk
+braze.com
```

Comment:
It is owned by Braze, they also provides services for other companies, such as HBO and Etsy.